plugins/CuraEngineBackend/CuraEngineBackend.py

```diff
@@ -658,7 +670,7 @@ def _onGCodePrefixMessage(self, message: Arcus.PythonMessage) -> None:
     ##  Creates a new socket connection.
     def _createSocket(self, protocol_file: str = None) -> None:
         if not protocol_file:
-            protocol_file = os.path.abspath(os.path.join(PluginRegistry.getInstance().getPluginPath(self.getPluginId()), "Cura.proto"))
+            protocol_file = os.path.abspath(os.path.join(str(PluginRegistry.getInstance().getPluginPath(self.getPluginId())), "Cura.proto"))
```

Comment:
Doing this you are forcing a possible `None` value to be `'None'` and in this case the next function will not work.

Reply:
Fixed in ac0192f